Scripts/Modules/SerializedPreferencesModule.cs

```diff
@@ -94,8 +95,10 @@ internal string SerializePreferences()
 				}
 
 				if (item != null)
-					preferences.items.Add(item);
+					payloads[serializer.GetType()] = item;
```

Comment:
This is there to prevent an error which occurs on deserialization if you do somehow save more than one payload of the same type. I also check against this in the tool, but I thought it was a good idea to leave the safeguard in place.

Should we also print a warning if something tries to serialize the same type twice?